unit-test/CMakeLists.txt

```diff
@@ -19,8 +19,10 @@ add_cfe_coverage_stubs("ds_internal"
 # Link with the cfe core stubs and unit test assert libs
 target_link_libraries(coverage-ds_internal-stubs ut_core_api_stubs ut_assert)
 
-# Include and expose unit test utilities includes
+# Include and expose unit test utilities, fsw/inc, and fsw/src includes
 target_include_directories(coverage-ds_internal-stubs PUBLIC utilities)
+target_include_directories(coverage-ds_internal-stubs PUBLIC ../fsw/inc)
+target_include_directories(coverage-ds_internal-stubs PUBLIC ../fsw/src)
```

Comment:
Minor nitpick here, these actually do not need to be separate calls to `target_include_directories` ... they can be all listed in one call.  Furthermore it might be more future-proof to reference the public includes via the ds target property, that is:

```
target_include_directories(coverage-ds_internal-stubs PUBLIC
    $<TARGET_PROPERTY:ds,INTERFACE_INCLUDE_DIRECTORIES>
)
```

This better follows the "DRY" principle - don't repeat yourself....  but it's not critical.  Unfortunately that won't work for "src" because there is no property for that one).

I'd also suggest ditching the "utilities" subdirectory here - doesn't seem like it accomplishes anything except adding some build complexity, but that can be considered as a future enhancement, not related to this change.